istio.VERSION

```diff
@@ -7,7 +7,7 @@ export MIXER_TAG="replace-by-actual-tag"
 export PILOT_HUB="gcr.io/istio-testing"
 export PILOT_TAG="replace-by-actual-tag"
 export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/replace-by-actual-tag/artifacts/istioctl"
-export PROXY_TAG="80ec4f6c2a672b00e55d34ef9b5d26a42a98da42"
+export PROXY_TAG="54077a0c8486b6db7b2fc5d10619b6813bb40767"
```

Comment:
why does this need to change in this pr?

istio.Version is in .gitignore, why the changes?